src/templates/minimal/src/server/index.js

```diff
@@ -0,0 +1,57 @@
+import express from 'express';
+import path from 'path';
+import fs from 'fs';
+import { isDev } from 'dev-toolkit/settings';
+
+const createHTML = ({ html, script }) => `<!DOCTYPE html>
+<html>
+  <body>
```

Comment:
Only further comment would be to have a `<head><title>..</title></head>` (making it the minimum to be functional in production)

Reply:
done 👍